style_guide.md

```diff
@@ -0,0 +1,76 @@
+# Jsonnet Style Guide
+This style guide documents how we want to handle jsonnet files when building the schemas with consistency. For more general principles and styling rules go to [Databricks repository](https://github.com/databricks/jsonnet-style-guide#databricks-jsonnet-guide).
+
+## Folder structure
+
+### England/Wales and Northern Ireland
+
+- Jsonnet source files are separated into two folders: `england-wales` and `northern-ireland` with a number of sub-folders, each for corresponding schemas (e.g. `ccs`, `communal-establishment`, `household`, `individual`). These are divided into blocks in `blocks` folder.
+
+### Top level Jsonnet files in these folders
+
+- For each schema there is a lists of block and placeholder imports, as well as schema structure divided into sections, named after given schema and ending with `.jsonnet` suffix.
+
+
+## Common Jsonnet
+
+### Lib folders
+
+- Rules and placeholders are located in `lib` folder in corresponding schemas along with `blocks` folder.
+
+### Rules and placeholders
+
+- These are separated into: `common_rules`, `placeholders` and `rules`. These files should end with `.libsonnet` suffix.
+
+### Top level and extended
+
+- `common_rules` and `placeholders` are located in top level "lib" folder.
+- `rules` files for each schema are located in schema's sub-folder and extend some of the top-level `common_rules`.
+
+## Global variables
+
+### Top level arguments
+
+- For each schema created it is possible to pass variables, e.g.: `region_code` and `census_month_year_date`, with different outputs being generated depending on their value.
```

Comment:
We should explain why variables like this exist. These are things that are known at questionnaire launch and won't change during the Census.

Reply:
I added some more to it, thanks!👍